api/services/elastigroup/azure/scaleSetVms/paths/roll.yaml

```diff
@@ -0,0 +1,65 @@
+get:
```

Comment:
@vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master